controllers/storagecluster/provider_server.go

```diff
@@ -210,6 +210,8 @@ func (o *ocsProviderServer) createService(r *StorageClusterReconciler, instance
 
 		instance.Status.StorageProviderEndpoint = fmt.Sprintf("%s:%d", nodeAddresses[0], ocsProviderServiceNodePort)
 
+	} else {
+		instance.Status.StorageProviderEndpoint = fmt.Sprintf("%s:%d", actualService.Spec.ClusterIP, ocsProviderServicePort)
 	}
```

Comment:
should use a switch case here instead of if-elseif-else?

Reply:
will make the change

Reply:
Done